tools/spark-block-cleaner/pom.xml

```diff
@@ -37,6 +37,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-util-scala_${scala.binary.version}</artifactId>
```

Comment:
seems redundant, since kyuubi-common depends on it

Reply:
Oh, good catch. Removed.